tests/unit/lms/models/grading_info_test.py

```diff
@@ -40,7 +40,7 @@ def test_it_enforces_non_nullable_field_presence(
         db_session.add(grading_info)
         with pytest.raises(
             sqlalchemy.exc.IntegrityError,
-            match=f'null value in column "{non_nullable_field}" violates not-null constraint',
+            match=f'null value in column "{non_nullable_field}"',
```

Comment:
We don't write this type of test very often we we could as well delete them but this seems like a good compromise.